ShenkinStore/Models/ShenkinContext.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+using ShenkinStore.Models;
+
+namespace ShenkinStore.Models
+{
+    public partial class ShenkinContext : DbContext
+    {
+        public ShenkinContext()
+        {
+        }
+
+        public ShenkinContext(DbContextOptions<ShenkinContext> options)
+            : base(options)
+        {
+        }
+        public virtual DbSet<Product> Products { get; set; }
+
+        //        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
Remove those comments

Reply:
As I saw on the tutorial, they recommended to leave it in comment because we might need it in the future.

Reply:
So add a comment above "A Comment in case we'll need for future use"